nyc_data/ppe/views.py

```diff
@@ -197,7 +197,7 @@ def post(self, request):
         if form.is_valid():
             try:
                 import_obj = data_import.handle_upload(
-                    request.FILES["file"], form.data["name"]
+                    request.FILES["file"], form.data["name"], form.data["data_current"]
```

Comment:
outta curiosity - what is form.data["data_current"] in reference to?

Reply:
It references this: https://github.com/RTCovid/nyc-ppe/pull/83/files#diff-3c097fd331943ddebfb8e0422457739b

A new field on the form to ask what date the data covers